pilot/pkg/proxy/envoy/v1/watcher.go

```diff
@@ -255,6 +257,14 @@ func NewProxy(config meshconfig.ProxyConfig, node string, logLevel string) proxy
 	}
 }
 
+// NewV2Proxy creates an instance of the proxy using v2 bootstrap
+func NewV2Proxy(config meshconfig.ProxyConfig, node string, logLevel string) proxy.Proxy {
```

Comment:
I think putting v2 into v1 envoy directory is against the intention of @rshriram to have two config generation engines.

Reply:
For now it's the simplest / cleanest solution - we can move later, but eventually the entire v1 directory, including this part, will go away. Once SDS is available we won't need any code here - and this will move to bootstrap.